### PR TITLE
Adds secret name in service binding status

### PR DESCRIPTION
**Pull request by DhritiShikhar**
Partially solves: https://jira.coreos.com/browse/ODC-1327

This PR exposes the secret name in service binding status.

ToDo:
- [x] Update README

### CODE REVIEW

**Comment by baijum** [comment]
Do we need this as it is always the name of CR.

**Comment by sbose78** [comment]
Yes
- to inform the user that we created a `secret`
- we might change the naming convention in future to accommodate conflicts.

File: README.md [hunk 1]
```diff
@@ -276,6 +276,7 @@ spec:
 
 | Field | Description |
 |-------|-------------|
-| Status | the binding status of Service Binding Request |
+| BindingStatus | The binding status of Service Binding Request |
+| Secret | The name of the intermediate secret |
```
Comment (sbose78) [inline README.md]: Thanks for this @DhritiShikhar . Could you also include a CR yaml in the readme which has the `status` populated?

Comment (DhritiShikhar) [reply README.md]: @sbose78 Yes. I am planning to do that. But I don't want to do it as a part of this PR. I will raise another PR for the changes in README.

**Pull request merged**